src/js/profile/mobile/widget/DatePicker.js

```diff
@@ -0,0 +1,1826 @@
+/* eslint-disable guard-for-in */
+/* eslint-disable block-scoped-var */
+/* eslint-disable no-eq-null */
+/* eslint-disable one-var */
+/* eslint-disable vars-on-top */
+/*global window, define, ns */
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ * @example
+ * <div class="ui-datepicker" id="pickers"></div>
+ *
+ * id="pickers" should be fixed.
+ *
+ * @since 6.0
+ * @class ns.widget.core.DatePicker
+ * @extends ns.widget.BaseWidget
+ * @author Dohyung Lim <delight.lim@samsung.com>
+ */
+(function (document, ns) {
+	"use strict";
+	//>>excludeStart("tauBuildExclude", pragmas.tauBuildExclude);
+	define(
+		[
+			"../../../core/engine",
+			"../../../core/util/DOM/attributes",
+			"../widget",
+			"./BaseWidgetMobile"
+		],
+		function () {
+			//>>excludeEnd("tauBuildExclude");
+			var utilsObject = ns.util.object,
+				engine = ns.engine,
+
+				DatePicker = function () {
+					this.options = utilsObject.merge({}, DatePicker.defaults);
+					this.observer = null;
+				},
+
+				defaults = {
+					value: false, // false is past selectable, true is past selection not possible.
+					type: "calendar" // calendar or wheel
+				},
+
+				BaseWidget = ns.widget.BaseWidget,
+				prototype = new BaseWidget();
+
+			DatePicker.prototype = prototype;
+			DatePicker.defaults = defaults;
+
+			prototype._init = function (element) {
+				var DatePickerObj;
+
+				if (typeof (DatePickerObj) == "undefined") {
+					DatePickerObj = {
+						newId: function () {
+							return this.lastId++;
+						}
+					};
+					DatePickerObj.isChrome = (navigator.userAgent.indexOf("Chrome") >= 0);
```

Comment:
I think we don't need to check the chrome.

Reply:
ok. I removed.

done.